### DISCUSSION TITLE
Roadmap of project

### DISCUSSION THREAD

**Discussion by askme23** [monkeytypegame/monkeytype#3967] [Questions] [answered]
Hey ) I really want to contribute to project, but don't know what may i start with. Is there a roadmap with ideas and features that weren't done yet or smth like that ?

**Comment by Miodec** [comment, marked as answer]
The only thing remotely close to a roadmap is just the list of issues on github. Feel free to comment on any of them if you wish to work on them. If you need a guide on getting started check the contributing file.

**Reply by askme23** [reply]
got you. i'll try to find something interesting for me
